config.siyuan.yaml

```diff
@@ -468,6 +468,13 @@ repos:
     interval: 7000
     name: ghcup
     <<: *oneshot_common
+  # elan
+  - type: shell_script
+    script: /worker-script/mirror-clone-v2.sh --workers 4 --target-type s3 --s3-prefix elan --s3-buffer-path /var/cache --print-plan 100 elan --target-mirror https://mirror.sjtu.edu.cn/elan --retain-elan-versions 2 --retain-lean-versions 20
```

Comment:
1. You should use the `--s3-scan-metadata` flag to preserve the original last modified date, enabling us to detect file changes in case the upstream modifies a file in place.
2. There's no need to add `--target-mirror` because your module doesn't have this option.